pkg/policies/engine/rego/rego.go

```diff
@@ -49,6 +51,16 @@ func (r *Rego) Verify(ctx context.Context, policy *engine.Policy, input []byte)
 		return nil, fmt.Errorf("failed to parse input: %w", err)
 	}
 
+	// add arguments
+	if args != nil {
```

Comment:
it's not clear to me if you are doing any processing for single, comma-separated or multiline now?

Reply:
This code is just embedding whatever comes as args (`map[string]any`) into the `input` object, so that it can be used as `input.args.foo` from the Rego code.

Reply:
but is it understanding that a multi-line or comma separated values are arrays?

Reply:
Yes, that parsing is done in the changes from policies.go. I've added a bunch of tests for all cases.